docs/resources/vpn_gateway.md

```diff
@@ -75,37 +94,62 @@ The following arguments are supported:
 
 * `name` - (Required, String) The name of the VPN gateway. Only letters, digits, underscores(_) and hypens(-) are supported.
 
-* `vpc_id` - (Required, String, ForceNew) The ID of the VPC to which the VPN gateway is connected.
+* `availability_zones` - (Required, List, ForceNew) The list of availability zone IDs.
```

Comment:
it's better to change the order in schema

Reply:
sure, the order is updated